exhibitor-standalone/src/main/resources/buildscripts/standalone/gradle/build.gradle

```diff
@@ -1,7 +1,17 @@
+version = '1.0'
```

Comment:
does the automatic version calculation only depend on the current group and version tags (exhibitor and 1.5.4 respectively)?